witness/src/Dscp/Snowdrop/Configuration.hs

```diff
@@ -256,8 +264,9 @@ data Proofs
 ----------------------------------------------------------------------------
 
 data ExpanderException =
-    MTxDuplicateOutputs
+      MTxDuplicateOutputs
     | CantResolveSender
+    | ExpanderInternalError String
```

Comment:
I personally don't think that internal errors should appear in `ExceptT` or be thrown with `throwM` as exceptions, `error` is better, because when I catch exceptions in upper logic I shouldn't care or know about internal exceptions. We even had this rule in cardano's exceptions guide.

Reply:
Not sure I understand you. We had `DBMalformedException` which was not an `error` thing.

Reply:
Hm, right, it took place.